README.md

```diff
@@ -4,6 +4,25 @@ A graph based query language on top of [dagger](https://github.com/google/dagger
 
 Dagger Query can be used to investigate the dagger dependency graph.
 
+
+## How to use Dagger Query project?
+
+1. Add `//src/com/google/daggerquery:dagger_query_plugin` target in a **plugins** attribute of your app's target.
+2. Go to `src/com/google/daggerquery/BUILD` and use **dagger_query** rule with your app's target name. 
```

Comment:
Should they be modifying the dagger_query rule inside project/src/com/google/daggerquery/BUILD?
or should they be using this rule in their own codebase?

Reply:
No, now it can be used anywhere! Thanks, I updated the second step in the README in the latest commit 😊